contao/core/2014-02-13.yaml

```diff
@@ -2,16 +2,10 @@ title:     PHP object injection vulnerability allows for arbitrary code executio
 link:      https://contao.org/en/news/major-security-hole-found-in-contao.html
 cve:       ~
 branches:
-    2.11.x:
```

Comment:
Why is a new CVE affecting information of a previous one here? Shouldn't it be all in a new file?